core/controlplane/config/templates/cluster.yaml

```diff
@@ -980,6 +980,19 @@ addons:
 #     rbac:
 #       enabled: true
 #
+#   # When set to true this configures the k8s aws provider so that it doesn't modify every node's security group to include an additional ingress rule per an ELB created for a k8s service whose type is "LoadBalancer"
+#   # It requires that the user has setup a rule that allows inbound traffic on kubelet ports 
+#   # from the local VPC subnet so that load balancers can access it.  Ref: https://github.com/kubernetes/kubernetes/issues/26670
+#   disableSecurityGroupIngress: false
+#   
+#   # Command line flag passed to the controller-manager. (default 40s)
+#   # This is the amount of time which we allow running Node to be unresponsive before marking it unhealthy. 
+#   # Must be N times more than kubelet's nodeStatusUpdateFrequency (default 10s).
+#   nodeMonitorGracePeriod: "40s"
+#   
+#   # Command line flag passed to kubelet. (default 10s)
+#   # Specifies how often kubelet posts node status to master. Note: be cautious when changing the constant, it must work with nodeMonitorGracePeriod in nodecontroller.
+#   nodeStatusUpdateFrequency: "10s"
```

Comment:
I guess you intended to add this comment about the `nodeStatusUpdateFrequency` setting, which is for node pools, under the `worker.nodePools[]` key but not under the `experimental` key?